modules/jena/src/main/scala/ch/epfl/bluebrain/nexus/rdf/jena/syntax/FromJenaSyntax.scala

```diff
@@ -0,0 +1,63 @@
+package ch.epfl.bluebrain.nexus.rdf.jena.syntax
+
+import ch.epfl.bluebrain.nexus.rdf.{Graph, Node}
+import org.apache.jena.rdf.model
+import org.apache.jena.rdf.model.{Model, RDFNode, Resource}
+
+trait FromJenaSyntax {
+  import ch.epfl.bluebrain.nexus.rdf.jena.{JenaConverters => conv}
+
+  implicit class AsRdfModel(model: Model) {
```

Comment:
I don't think I understand the logic behind the class names `AsRdfModel`. Wouldn't something like `AsRdfGraph` make more sense here?

Reply:
There are naming collisions if we change the naming based on the target. Maybe inverting would be better: `ModelAsRdf`?

Reply:
that sounds better.